androidApp/onbording/src/main/java/tech/takahana/iconwallpaper/android/onbording/ui/components/Announcements.kt

```diff
@@ -0,0 +1,32 @@
+package tech.takahana.iconwallpaper.android.onbording.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun Announcements(message: String) {
+    Box(
+        modifier = Modifier
+            .fillMaxWidth()
+            .height(60.dp)
+            .background(color = Color.White),
+        contentAlignment = Alignment.Center,
+    ) {
+        Text(message)
+    }
+}
+
+@Preview(showSystemUi = true)
```

Comment:
`showSystemUi = true` の方だと、Boxが全画面に広がって見えるので、高さが分かりづらいかもしれないです👀
`showSystemUi = true` は不要かも知れません。
（スクショは背景色を赤色にしています）

![スクリーンショット 2022-01-08 16 50 44](https://user-images.githubusercontent.com/22670961/148636597-6b600cdd-4750-4992-8b49-dbf152518a72.png)

```suggestion
@Preview
```